Tests/AWSLambdaRuntimeCoreTests/MockLambdaServer.swift

```diff
@@ -30,7 +30,7 @@ internal final class MockLambdaServer {
     private var shutdown = false
 
     public init(behavior: LambdaServerBehavior, host: String = "127.0.0.1", port: Int = 7000, keepAlive: Bool = true) {
-        self.group = MultiThreadedEventLoopGroup(numberOfThreads: System.coreCount)
```

Comment:
This makes Xcodes debug view much easier to deal with! And we only need a single core for the MockServer anyway.